### PR TITLE
Sage dumb

### PR DESCRIPTION
**Pull request by dragynmada**
Bring networking interface into eye tracking

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -17,6 +19,8 @@
         android:theme="@style/Theme.OpenCVTest">
         <activity
             android:name=".MainActivity"
+            android:configChanges="orientation|screenSize|keyboardHidden"
+            android:exported="true">
```
Comment (RebornChampion) [inline app/src/main/AndroidManifest.xml:23]: is this a goofy merge thing? duplicating the android:exported="true">

File: app/src/main/java/ece/wisc/opencvtest/EyeDetection.java [hunk 2]
```diff
@@ -519,6 +529,21 @@ public void relearnFace(View v)
 
     public void setOffsets(double[][] offsets) {
         this.offsets = offsets;
-        this.sendBT = true;
+    }
+
+    public int getMouse_x() {
```
Comment (RebornChampion) [inline app/src/main/java/ece/wisc/opencvtest/EyeDetection.java:534]: I don't think we ended up actually using these? maybe we should be or get rid of them?

**Review by RebornChampion — COMMENTED** [review]
2 comments, hoping this submit review shows you them. not sure how pr's work on github

File: app/src/main/java/ece/wisc/opencvtest/EyeDetection.java (hunk 2 above)
Comment (dragynmada) [reply app/src/main/java/ece/wisc/opencvtest/EyeDetection.java:534]: They could be deleted, for now it's fine if they aren't used

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (dragynmada) [reply app/src/main/AndroidManifest.xml:23]: I have no clue

**Review by RebornChampion — APPROVED** [review]

**Pull request merged**